src/OneExplorer/OneExplorer.ts

```diff
@@ -463,7 +463,10 @@ export class OneNode extends vscode.TreeItem {
     //
     // However, resourceExtname returns info of vscode Explorer view (not of OneExplorer).
     //    "when": "view == OneExplorerView && viewItem == config"
-    this.contextValue = node.typeAsString;
+    //
+    // To resolve above the issue, it appends the extname to the contextValue.
```

Comment:
The above comment is not the issue but the history. It's to explain why we should use 'viewItem'.